pythonforandroid/recipes/hostpython2/__init__.py

```diff
@@ -1,60 +1,18 @@
+from pythonforandroid.python import HostPythonRecipe
 
-from pythonforandroid.toolchain import Recipe, shprint, current_directory, info, warning
-from os.path import join, exists
-import os
-import sh
 
+class Hostpython2Recipe(HostPythonRecipe):
```

Comment:
Love how much simpler it looks now! :heart: (assuming it actually still works)

Reply:
It does work ;)